lang/java/reef-common/src/main/java/org/apache/reef/driver/evaluator/EvaluatorRequestor.java

```diff
@@ -41,5 +40,5 @@
    * Get a new Builder for the evaluator with fluid interface.
    * @return Builder for the evaluator
    */
-  EvaluatorRequestorImpl.Builder newRequest();
+  EvaluatorRequest.Builder newRequest();
```

Comment:
This looks like a bug fix which could be its own JIRA / PR.

Reply:
Is this required? I ask because the current PR heavily relies on having this bug fixed.